app/src/main/res/layout/item_card_events.xml

```diff
@@ -21,7 +21,6 @@
             android:layout_gravity="end"
             android:layout_marginTop="@dimen/fab_margin_top"
             android:layout_marginRight="80dp"
-            android:scaleType="center"
```

Comment:
Try app:scaleType

Reply:
It is showing error. `Android resource linking failed`